style.css

```diff
@@ -128,3 +131,263 @@ h1 {
   background-position: bottom right;
   background-repeat: no-repeat;
 }
+
+.grid {
```

Comment:
- [ ] You need to add the paragraph for the first project and align the ```See Project``` button to the left, please stick to the Figma design as much as possible

|Your design|Figma design|
|--|--|
|![Screenshot_20220527_192857](https://user-images.githubusercontent.com/25227022/170769989-94d147ab-a7b9-43d4-851b-f781358397d3.png)|![Screenshot_20220527_192956](https://user-images.githubusercontent.com/25227022/170770023-d654aee1-f62d-455d-a073-86d9aa0b5b16.png)|